### PR TITLE
Vmcommon storage updates

### PR DESCRIPTION
**Pull request by sasurobert**
Vmcommon storage updates

### CODE REVIEW

File: cmd/node/config/config.toml [hunk 1]
```diff
@@ -707,6 +707,9 @@
         # If set to 0, then MaxUInt64 will be used
         MaxGasPerVmQuery = 1500000000  #1.5b
 
+[BuiltInFunctions]
+    AutomaticCrawlerAddress = "erd1fpkcgel4gcmh8zqqdt043yfcn5tyx8373kg6q2qmkxzu4dqamc0swts65c"
```
Comment (iulianpascalau) [inline cmd/node/config/config.toml:711]: please move this in systemSmartContractsConfig.toml

**Review by raduchis — DISMISSED** [review]

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
please update the esdt storer stub, as the tests are failing:
```
node/options_test.go#L285

cannot use esdtStorer (variable of type *"github.com/ElrondNetwork/elrond-go/node/mock".EsdtStorageHandlerStub) as type vmcommon.ESDTNFTStorageHandler in argument to WithESDTNFTStorageHandler
```

**Review by iulianpascalau — CHANGES_REQUESTED** [review]
.

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — DISMISSED** [review]
System test passed.

File: node/mock/esdtStorageHandlerStub.go [hunk 2]
```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/data"
 	"github.com/ElrondNetwork/elrond-go-core/data/esdt"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"math/big"
```
Comment (bogdan-rosianu) [inline node/mock/esdtStorageHandlerStub.go:7]: imports are not sorted

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: node/mock/esdtStorageHandlerStub.go (hunk 2 above)
Comment (sasurobert) [reply node/mock/esdtStorageHandlerStub.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**